pytest_fixtures/component/os.py

```diff
@@ -17,7 +19,7 @@ def default_os(
     """
     os = getattr(request, 'param', None)
     if os is None:
-        search_string = 'name="RedHat" AND (major="6" OR major="7" OR major="8")'
+        search_string = constants.DEFAULT_OS_SEARCH_QUERY
```

Comment:
FYI, there is a [discussion/PR](https://github.com/SatelliteQE/robottelo/pull/14670#discussion_r1567443179) in progress on how we retrieve the default OS.

We are to remove the need of search string especially for searching default os.

Reply:
@jyejare I don't see much progress on the discussion there. Do we want to hold off on this PR/fix? Is someone already assigned to work on it?

Reply:
@jameerpathan111 @Gauravtalreja1  can either choose to fix [discussion/PR](https://github.com/SatelliteQE/robottelo/pull/14670#discussion_r1567443179) comment in this PR or merge this as is if its urgent.

Which we would you like to go ?

Reply:
a ping would have been nice

Reply:
> @jyejare I don't see much progress on the discussion there. Do we want to hold off on this PR/fix? Is someone already assigned to work on it?

2 weeks ago there were two approvals and no answer from @lpramuk requested changes